File: dotnetweb-dest/MyService/OKS.cs
using System.Collections.Generic;
namespace MyService
{
    public class OKS
    {
        public virtual string __reflection { get; set; } = "MyService.OKS";
        public MyService.OKS test { get; set; }
        public MyService.OKS[] okss { get; set; }
        public System.Collections.Generic.List<MyService.OKS> cheese { get; set; }
        public System.Collections.Generic.Dictionary<string, MyService.OKS> use { get; set; }
    }
}

File: pywebsocket/MyService/OKS.py
from __future__ import annotations
from typing import Dict, List, TypedDict

class OKS(TypedDict):
    def __init__(self):
        super().__init__()
        self.__reflection = 'MyService.OKS'

    __reflection: str
    test: OKS
    okss: List[OKS]
    cheese: List[OKS]
    use: Dict[str, OKS]